bin/Powheg/run_pwg.py

```diff
@@ -679,10 +684,6 @@ def runGetSource(parstage, xgrid, folderName, powInputName, process, tagName) :
   sed -e 's/PROCESS/'${process}'/g' ${WORKDIR}/runcmsgrid_powheg.sh > runcmsgrid.sh
 fi
 
-
-sed -i s/SCRAM_ARCH_VERSION_REPLACE/${SCRAM_ARCH}/g runcmsgrid.sh
```

Comment:
could it be that you started modifying the script before updating to the central repository?
I recently introduced these lines to ensure the back-/forward- compatibility of the gridpacks in production,
unless there is a specific reason to remove them they need to be there :-)